source/package.json

```diff
@@ -28,6 +28,6 @@
   "dependencies": {
     "@digipolis-gent/modal": "^1.0.4",
     "fs-extra": "^11.1.0",
-    "gent_styleguide": "^6.0.0-alpha10"
+    "gent_styleguide": "^6.0.0-alpha15"
```

Comment:
De lock file is niet aangepast, waardoor de nieuwe styleguide niet geladen zal worden. Is dit aangepast via `npm update gent_styleguide`?